frontend/packages/process-editor/src/ProcessEditor.tsx

```diff
@@ -28,7 +31,10 @@ export const ProcessEditor = ({
 
   return (
     <BpmnContextProvider bpmnXml={bpmnXml}>
-      <Canvas onSave={onSave} appLibVersion={appLibVersion} />
+      <div className={classes.container}>
+        <Canvas onSave={onSave} appLibVersion={appLibVersion} />
+        <ConfigPanel appLibVersion={appLibVersion} />
```

Comment:
What do you think of adding the `appLibVersion` to the `BpmnContextProvider` and then exporting the state for `isEditAllowed`? The reason I'm asking is that we now passing the `appLibVersion `prop to both `ConfigPanel` and `Canvas` and in both places, we need to invoke the `supportsProcessEditor(appLibVersion)`. By centralizing this we do not need to care about the implementation of this within the components, instead just get the `isEditAllowed` directly from the `BpmnContextProvider`.

```typescript
const { bpmnXml, isEditAllowed } = useBpmnContext();
```

Not sure if I communicated what I meant, but please do not hesitate to ask 😅

Reply:
Great suggestion! This got rid of some duplicated code 😊